_src/static/sass/utilities/_variables.scss

```diff
@@ -3,156 +3,316 @@
+
+// font families
+$font-family-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
+
+// scale roots
+$font-scale-small: 14;
+$font-scale-medium: 16;
+$font-scale-large: 16;
+$font-scale-base: $font-scale-medium;
+
+$em-base: $font-scale-medium; // deliberately sets bourbon-based em-base (http://bourbon.io/docs/#em-base)
+
+$font-sizes: (
+    base:           $font-scale-base,
+    mid-small:      ceil(golden-ratio($font-scale-base, -1)),
+    small:          ceil(golden-ratio($font-scale-base, -2)),
+    x-small:        ceil(golden-ratio($font-scale-base, -3)),
+    xx-small:       ceil(golden-ratio($font-scale-base, -4)),
+    mid-large:      ceil(golden-ratio($font-scale-base, 1)),
+    large:          ceil(golden-ratio($font-scale-base, 2)),
+    x-large:        ceil(golden-ratio($font-scale-base, 3)),
+    xx-large:       ceil(golden-ratio($font-scale-base, 4)),
+);
+
+// line-height scale
+$line-heights: (
+    none:           0,
+    base:           1.25,
+    mid-small:      1,
+    small:          0.75,
+    x-small:        0.75,
+    mid-large:      1.25,
+    large:          1.48,
+    x-large:        1.48
+);
+
+// size scale
+$font-weights: (
+    normal:         400,
+    light:          300,
+    x-light:        200,
+    semi-bold:      600,
+    bold:           700,
+    x-bold:         800
+);
 
 // ----------------------------
-// #Colors (Application)
+// #DEPTH
 // ----------------------------
-$color-draft:             $gray-lightest;
-$color-live:              $blue;
-$color-ready:             $green;
-$color-warning:           $orange-lighter;
-$color-error:             $red-lighter;
-$color-staff-only:        $black;
-$color-visibility-set:    $black;
+$z-depths: (
+    base:           1,
+    mid-back:       -10,
+    back:           -100,
+    very-back:      -1000,
+    mid-front:      10,
+    front:          100,
+    very-front:     1000
+);
+
+$spacing-v: (
+    base:       $baseline-v,
+    mid-small:  ($baseline-v*0.75),
+    small:      ($baseline-v/2),
+    x-small:    ($baseline-v/4),
+    mid-large:  ($baseline-v*1.5),
+    large:      ($baseline-v*2),
+    x-large:    ($baseline-v*4)
+);
 
-$color-heading-base:       $gray-darker;
-$color-copy-base:          $gray-light;
-$color-copy-emphasized:    $gray-darker;
+$spacing-h: (
+    base:       $baseline-h,
+    mid-small:  ($baseline-h*0.75),
+    small:      ($baseline-h/2),
+    x-small:    ($baseline-h/4),
+    xx-small:   ($baseline-h/8),
+    mid-large:  ($baseline-h*1.5),
+    large:      ($baseline-h*2),
+    x-large:    ($baseline-h*4)
+);
 
-$link:            $blue;
-$link-hover:      $blue-lighter;
-$link-visited:    $blue-dark;
 
+// ----------------------------
+// #TIMING
+// ----------------------------
+$timing: (
+    base:       0.75s,
```

Comment:
Good question - I'd prefer us to use them some and see how many we end up referencing. I know in the past, the slower ones have been used for larger UI animations/transitions while the very fast ones are used for basic transitions. Let's revisit as we get further in PL work.